e2e/flags/.snapshots/TestInitCommand

```diff
@@ -13,12 +13,12 @@ rule:
 scan:
     context: ""
     data_subject_mapping: ""
-    debug: false
```

Comment:
Can we keep that one (deprecated) and make it pass the right value to the new `log-level` instead?

Reply:
I think it should only be omitted from generating the default config, but still supported for existing files.